Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/PID.mo

```diff
@@ -27,23 +27,24 @@ block PID "Identify control gain, integral time, and derivative time of the PID
     final quantity="Time",
     final unit="s",
     min=100*Buildings.Controls.OBC.CDL.Constants.eps)
-    "Connector for time constant signal for the integral term"
+    "Time constant signal for the integral term"
     annotation (Placement(transformation(extent={{100,-20},{140,20}})));
   Buildings.Controls.OBC.CDL.Interfaces.RealOutput Td(
     final quantity="Time",
     final unit="s",
     min=100*Buildings.Controls.OBC.CDL.Constants.eps)
-    "Connector for time constant signal for the derivative term"
+    "Time constant signal for the derivative term"
     annotation (Placement(transformation(extent={{100,-80},{140,-40}}),
         iconTransformation(extent={{100,-90},{140,-50}})));
-  Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO.PIDGain gai
-    "Calculate the control gain"
+protected
+  BaseClasses.PIDGain                                                        gai
```

Comment:
full path, one line.

Reply:
The full path is used and the format is fixed.